src/Mod/Part/App/TopoShape.cpp

```diff
@@ -372,20 +372,19 @@ void TopoShape::convertToMatrix(const gp_Trsf& trsf, Base::Matrix4D& mtrx)
 #if OCC_VERSION_HEX >= 0x070000
     gp_Mat m = trsf.VectorialPart();
     gp_XYZ p = trsf.TranslationPart();
-    Standard_Real scale = trsf.ScaleFactor();
 
     // set Rotation matrix
-    mtrx[0][0] = scale * m(1,1);
-    mtrx[0][1] = scale * m(1,2);
-    mtrx[0][2] = scale * m(1,3);
+    mtrx[0][0] = m(1,1);

```

Comment:
Removing the scale factor here makes this function inconsistent for OCC < 7.0 and >= 7.0
So, what is the reason of removing it?